site/en/workflows.md

```diff
@@ -14,14 +14,14 @@ There are several steps necessary to get the fulltext of a scanned print. The wh
 
 The following instruction describes all steps of the OCR workflow. Depending on your particular print or rather images not all of those steps will be necessary to obtain good results. Whether a step is required or optional is indicated in the description of each step. 
 
-## Image Optimization
-Prepare image for better OCR.
+## Image Optimization (on Page Level)
+At first, the image should be prepared for OCR.
 
 
```

Comment:
Agreed. What concrete steps can we propose here? `ocrd_im6convert` is currently hard to recommend over image magick cmdline or similar if the user knows what steps make sense for the data. But I struggle to give a general recommendation to explore pre-(pre-)processing for possibly better results in certain cases... @VolkerHartmann since you've been benchmarking All The Workflows: Have you experimented with pre-binarization image processing and iff so can you share your experience?

Reply:
> What concrete steps can we propose here?

I would at least document the fact that this can be vital.

> `ocrd_im6convert` is currently hard to recommend over image magick cmdline or similar if the user knows what steps make sense for the data.

What do you mean hard to recommend? It's the only off the shelf solution we can offer. You can see some examples in the ocrd-tool description.

> But I struggle to give a general recommendation to explore pre-(pre-)processing for possibly better results in certain cases

I have experimented a little with the `textcleaner` script from Fred Weinhaus, using

    textcleaner -g -e normalize -f 25 -o 2 -t 30

That is:
1. grayscale conversion 
2. normalize brightness
3. noise filtering
4. text smoothing

which is then equivalent to `-respect-parenthesis -colorspace gray -type grayscale -normalize \( -clone 0 -colorspace gray -negate -lat 25x25x2% -contrast-stretching 0 -blur 1x65535 -level 30x100% \) -compose copy_opacity -composite -fill white -opaque none -alpha off`.

But obviously, the filter param constants I chose may depend on the input's pixel resolution and script height.